developers/weaviate/client-libraries/python/async.md

```diff
@@ -0,0 +1,196 @@
+
+When using the async helper functions, you must call the async `.connect()` method to connect to the server, and call `.close()` before exiting to clean up. (Except when using a [context manager](#context-manager).)
+
+The async helper functions take the same parameters for [external API keys](./index.md#external-api-keys), [connection timeout values](./index.md#timeout-values) and [authentication details](./index.md#authentication).
+
+<Tabs groupId="languages">
+<TabItem value="wcd" label="WCD">
+
+<FilteredTextBlock
+  text={PythonCode}
+  startMarker="# AsyncWCDInstantiation"
+  endMarker="# END AsyncWCDInstantiation"
+  language="py"
+/>
+
+</TabItem>
+<TabItem value="local" label="Local">
+
+  <FilteredTextBlock
+    text={PythonCode}
+    startMarker="# AsyncLocalInstantiationBasic"
+    endMarker="# END AsyncLocalInstantiationBasic"
+    language="py"
+  />
+
+</TabItem>
+
+<!-- TODO - add embedded equivalent when available in client -->
+
+<TabItem value="custom" label="Custom">
+
+<FilteredTextBlock
+  text={PythonCode}
+  startMarker="# AsyncCustomInstantiationBasic"
+  endMarker="# END AsyncCustomInstantiationBasic"
+  language="py"
+/>
+
+</TabItem>
+</Tabs>
+
+### Explicit instantiation
+
+If you need to pass custom parameters, use the `weaviate.WeaviateAsyncClient` class to instantiate a client. This is the most flexible way to instantiate the client object.
+
+When you instantiate a connection directly, you have to call the (now async) `.connect()` method to connect to the server.
+
+<FilteredTextBlock
+  text={PythonCode}
+  startMarker="# AsyncDirectInstantiationFull"
+  endMarker="# END AsyncDirectInstantiationFull"
+  language="py"
+/>
+
+## Sync and async methods
+
+The async client object is designed to be used in an `async` function or in an [`asyncio` event loop](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio-event-loop).
+
+Accordingly, a majority of the client methods are `async` functions that return [`Coroutines` objects](https://docs.python.org/3/library/asyncio-task.html#coroutine). However, some methods are synchronous and can be used in a synchronous context.
+
+As a rule of thumb, a method that involves a request to Weaviate is likely to be an async function, while a method that executes in a local context is likely to be synchronous.
+
+### How to identify async methods
+
+Async methods are identified by its method signature. Async methods are defined with the `async` keyword, and they return a `Coroutines` object.
+
+To see a method signature, you can use the `help()` function in Python, or use an IDE that supports code completion such as [Visual Studio Code](https://code.visualstudio.com/docs) or [PyCharm](https://www.jetbrains.com/help/pycharm/viewing-reference-information.html).
+
+### Example async methods
+
+Methods that involve a request to Weaviate is likely to be an async function. For example, each of the following operations is an async function:
```

Comment:
- are likely to be async functions
- same as above with stronger `likely` -> `will` language?